.github/workflows/build.yml

```diff
@@ -23,9 +30,23 @@ jobs:
       - name: Build CLI
         working-directory: ./packages/cli
         run: tsc -b
+      - name: Set the template version
+        run: |
+          echo "UI_TEMPLATE_VERSION=$(node scripts/get-next-test-version.js)" >> $GITHUB_ENV
+      - name: Release test versions
+        run: |
+          echo "E2E tests should target template version ${{ env.UI_TEMPLATE_VERSION }}"
+          git config --global user.email action@github.com
+          git config --global user.name GitHub Action
+        # node_modules/.bin/lerna version ${{ env.UI_TEMPLATE_VERSION }} --no-changelog --no-push -y
+        # echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }} > ~/.npmrc"
+        # Uncomment when ready
+        # npm run npm:publish:template
+        # node scripts/wait-for-published-packages.js -v ${{ env.UI_TEMPLATE_VERSION }}
```

Comment:
I need to make sure the action works as expected before uncommenting these lines

Reply:
Is this PR ready for review? 🤔

Reply:
yes. I need your input before uncommenting these lines.